compiler/daml-extension/BUILD.bazel

```diff
@@ -3,77 +3,65 @@
 
 package(default_visibility = ["//visibility:public"])
 
-load("@npm_bazel_typescript//:index.bzl", "ts_library")
-load("@build_bazel_rules_nodejs//:defs.bzl", "npm_package")
-load("//bazel_tools:pkg.bzl", "pkg_tar")
-
-ts_library(
-    name = "daml_extension_lib",
-    # TODO(MH): Unfortunately, the current packaging setup does not work with
-    # multiple source files. We need to figure out how to split this file up
-    # before it gets too big and package it properly.
-    srcs = ["src/extension.ts"],
-    node_modules = "@daml_extension_deps//:node_modules",
-    tsconfig = ":tsconfig.json",
-    deps = [
-        "@daml_extension_deps//@types",
-        "@daml_extension_deps//vscode",
-        "@daml_extension_deps//vscode-languageclient",
-    ],
+sh_test(
```

Comment:
That looks like you already added it in #2103

Reply:
Yes sorry bit of a mixup between branches.